src/transport/raw/UDP.cpp

```diff
@@ -103,6 +107,12 @@ CHIP_ERROR UDP::SendMessage(const Transport::PeerAddress & address, System::Pack
 
 void UDP::OnUdpReceive(Inet::IPEndPointBasis * endPoint, System::PacketBufferHandle && buffer, const Inet::IPPacketInfo * pktInfo)
 {
+    char addrBuffer[Transport::PeerAddress::kMaxToStringSize];
+    pktInfo->SrcAddress.ToString(addrBuffer, sizeof(addrBuffer));
+
+    ChipLogDetail(Inet, "UDP::OnUdpReceive message from %s", addrBuffer);
```

Comment:
This will fail to compile (with errors about unused variables) when detail logging is disabled....

Reply:
addressed in latest changes